### DISCUSSION TITLE
new Version 3 Problem with dynamic data

### DISCUSSION THREAD

**Discussion by FFKite** [MiczFlor/RPi-Jukebox-RFID#1911] [Q&A]
Hallo,

I did a fresh install of version 3. the problem is he loads the webface but no dynamic data. he only writes "Loading". 
it may be that some access rights are missing?

Thank you sorry for me English
Stefan

**Comment by AlvinSchiller** [comment]
Which browser do you use?
Firefox is currently not supported -> https://rpi-jukebox-rfid.readthedocs.io/en/latest/issues.html
> The Web UI will not work with Firefox, due to an issue with websockets and pyzmq. Please use a different browser for now
